java/odsa-spring/src/main/resources/application.properties

```diff
@@ -0,0 +1,21 @@
+# https://www.oracle.com/database/technologies/getting-started-using-jdbc.html
+spring.datasource.url=<URL>
+spring.datasource.username=<USERNAME>
+spring.datasource.password=<PASSWORD>
+
+# Properties for UCP - Universal Connection Pool (UCP). Spring Boot 2.4.0 or above is required
+spring.datasource.driver-class-name=oracle.jdbc.OracleDriver
+spring.datasource.type=oracle.ucp.jdbc.PoolDataSource
+spring.datasource.oracleucp.connection-pool-name=connectionPoolName1
+spring.datasource.oracleucp.initial-pool-size=15
+spring.datasource.oracleucp.min-pool-size=10
+spring.datasource.oracleucp.max-pool-size=30
+spring.datasource.oracleucp.connection-factory-class-name=oracle.jdbc.pool.OracleDataSource
+
+# Enable JPA / Hibernate debugging
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.format_sql=true
+logging.level.org.hibernate.SQL=DEBUG
+logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
```

Comment:
Do we really need debugging for a demo script? One would assume that a developer understands when/how to turn on debugging.

Reply:
adjustment accepted, debugging properties removed. @Kuassim please review, thanks